server/validation/contactSchema.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  number, object, string,
+} from 'yup';
+
+const contactSchema = object({
+  name: string().required('Name is required'),
+  email: string().email('Your email must be a valid email').required('Your email is required to contact us'),
+  message: string().required('Your message is required'),
+  phone: number().min(10, 'Your phone number must be 10 digits at least').required('Your phone number is required'),
```

Comment:
Same as I said at #54 , I believe this would compare the phone as a number if its larger than 10 not it's length.